test/Kripke-v1.1/Kripke-v1.1-RAJA/Kripke/Timing.cpp

```diff
@@ -108,6 +114,9 @@ void Timing::stop(std::string const &name){
     HPM_Stop(name.c_str());
 #endif
 
+#ifdef RAJA_USE_CALIPER
+    cali::Annotation(name.c_str()).end();
+#endif
```

Comment:
Yeah, this is orthogonal to the timing in RAJA itself, this is mainly meant to just work in the given applications so we can get numbers as we develop. Once RAJA has timings we should just get rid of this stuff (at least not use it for the purpose of continuous RAJA integration). More tomorrow, out for the day